### PR TITLE
Updating with changes dropped in 1830

### PR DESCRIPTION
**Pull request by moxiegirl**
Carry comments not incorporated into #1830 

Signed-off-by: Mary Anthony <mary@docker.com>

/cc @ehazlett @nathanleclaire

### CODE REVIEW

File: docs/install-machine.md [hunk 1]
```diff
@@ -40,6 +45,12 @@ To install on Linux, do the following:
 
 			$ docker-machine -v
 			machine version 0.4.0
+			
+## Install from binary
+
+You can install from the  binaries for the various platforms and architectures
+are available at the [Github
+Release](https://github.com/docker/machine/releases/) page.
```
Comment (dmp42) [inline docs/install-machine.md]: Maybe this sentence can be rephrased? 
`You can install from the  binaries for the various platforms and architectures are available at the`

File: docs/install-machine.md [hunk 2]
```diff
@@ -11,18 +11,23 @@ weight=3
...
-instruction or <a href="https://docs.docker.com/installation/windows" target="_blank">Windows installation</a> instructions.
+On OS X and Windows, Machine is installed along with other Docker products when
+you install the Docker Toolbox. For details on installing Docker Toolbox, see
+the <a href="https://docs.docker.com/installation/mac/" target="_blank">Mac OS X
+installation</a> instructions or <a
+href="https://docs.docker.com/installation/windows" target="_blank">Windows
+installation</a> instructions.
+
+If you aren't interested in installing the entire Docker Toolbox, you can install only
+Docker Machine using by [installing the binaries directly](https://github.com/docker/machine/releases/).
```
Comment (dmp42) [inline docs/install-machine.md]: `using by installing` can probably be rephrased

**Comment by moxiegirl** [comment]
@dmp42 Thank you for the commentary. Incorporated.

File: docs/install-machine.md [hunk 3]
```diff
@@ -41,9 +45,28 @@ To install on Linux, do the following:
...
+
+The Docker Machine team compiles binaries for several platforms and architectures and makes them available from [the Machine release page on Github](https://github.com/docker/machine/releases/). To install from a binary:
+
+1. Download the binary you want.
+2. Rename the binary to `docker-machine`.
+3. Move the `docker-machine` file to an appropriate directory on your system.
+
+    For example, on an OS X machine, you might move it to the `/usr/local/bin` directory.
+
+4. Ensure the files executable permssions are correct.
```
Comment (dmp42) [inline docs/install-machine.md]: `permssions ` -> `permissions`

**Comment by dmp42** [comment]
Thanks @moxiegirl 

One last nit - LGTM otherwise.

**Comment by moxiegirl** [comment]
Done!

**Comment by moxiegirl** [comment]
@ehazlett @nathanleclaire One more OK and we can merge this.  I can also cp into docs and that change can go live later this week.

**Comment by nathanleclaire** [comment]
Can you wrap to 80 columns please?

**Comment by moxiegirl** [comment]
@nathanleclaire done

File: docs/install-machine.md [hunk 4]
```diff
@@ -41,9 +49,31 @@ To install on Linux, do the following:
...
+Github](https://github.com/docker/machine/releases/). To install from a binary:
+
+1. Download the binary you want.
+2. Rename the binary to `docker-machine`.
+3. Move the `docker-machine` file to an appropriate directory on your system.
+
+    For example, on an OS X machine, you might move it to the `/usr/local/bin`
+    directory.
+
+4. Ensure the files executable permissions are correct.
```
Comment (nathanleclaire) [inline docs/install-machine.md]: Should this be `file's` (with apostrophe)?

**Comment by moxiegirl** [comment]
ping @nathanleclaire  good catch

**Comment by nathanleclaire** [comment]
Thanks!

LGTM

**Pull request merged**